plugin/helpers_bots.go

```diff
@@ -231,9 +231,21 @@ func (p *HelpersImpl) ensureBot(bot *model.Bot) (retBotID string, retErr error)
 		return "", errors.Wrap(kvGetErr, "failed to get bot")
 	}
 
-	// If the bot has already been created, there is nothing to do.
+	// If the bot has already been created, use it
 	if botIDBytes != nil {
 		botID := string(botIDBytes)
+
+		// ensure existing bot is synced with what is being created
+		botPatch := &model.BotPatch{
+			Username:    &bot.Username,
+			DisplayName: &bot.DisplayName,
+			Description: &bot.Description,
+		}
+
+		if _, patchBotErr := p.API.PatchBot(botID, botPatch); patchBotErr != nil {
+			return "", errors.Wrap(patchBotErr, "failed to patch bot")
```

Comment:
Given that the context of the `error` is clear, does it make sense to just go with the standard naming convention? 
```suggestion
		if _, err := p.API.PatchBot(botID, botPatch); err != nil {
			return "", errors.Wrap(err, "failed to patch bot")
```

Reply:
@hanzei yes, that makes sense. Fixed!